### DISCUSSION TITLE
Single page navigation with highlight

### DISCUSSION THREAD

**Discussion by shibusahu30** [ant-design/ant-design#37600] [Q&A]
![image](https://user-images.githubusercontent.com/44725484/190841490-f25017c3-06e4-46d1-84d5-20344d8ba173.png)

page scroll should highlight the corresponding item from navbar

How can I implement this type of interface using ANTD?

**Comment by fnoopv** [comment]
hey, `antd` is ui library, you need [this](https://alikhallad.com/how-to-highlight-active-links-for-one-page-navigation-with-javascript/)

**Reply by shibusahu30** [reply]
Thanks, will check this
